src/main/java/com/example/KitchenProject/CookBook.java

```diff
@@ -0,0 +1,15 @@
+package com.example.KitchenProject;
```

Comment:
Consider renaming the package to `com.example.kitchenproject`. It's a convention that package names are lowercase only.